tests/providers/google/cloud/transfers/test_gdrive_to_gcs.py

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from unittest import mock
+
+from airflow.providers.google.cloud.transfers.gdrive_to_gcs import GoogleDriveToGCSOperator
+
+FILE_ID = "1NHDk661aHxwpnvfVI2FbiFGVOM89FJTI"
+BUCKET = "destination_bucket"
```

Comment:
```suggestion
BUCKET = os.environ.get("GCP_GDRIVE_TO_GCS", "gdrive-to-gcs-bucket")
```
Since example dags are being run in system tests they should me made in a way that some variables (such bucket name) could be customized. It will allow other users run them without modifying code (goal is to run system test automatically some day).